gips/data/landsat/landsat.py

```diff
@@ -1346,3 +1390,203 @@ def _readraw(self, asset_type):
 
         verbose_out('%s: read in %s' % (image.Basename(), datetime.now() - start), 2)
         return image
+
+    def sentinel2_coreg_export(self, tmpdir, utm_zone):
+        landsat_shp = landsatRepository.get_setting('tiles')
+        sentinel2Data = import_data_class('sentinel2')
+        sentinel2Asset = sentinel2Data.Asset
+        sentinel2Repository = sentinel2Asset.Repository
+        extent = SpatialExtent.factory(sentinel2Data, site=landsat_shp, where="pr = '%s'" % self.id, pcov=33.0)
+        tiles = extent[0].tiles
+        date = datetime(self.date.year, self.date.month, self.date.day)
+        results = sentinel2Asset.query_service('L1C', tiles[0], date)
```

Comment:
All things related to fetching should be done by creating a `DataInventory` (either explicitly constructing or using the `sentinel2Data.inventory` factory), so that the ORM is happily automagically updated.  For now, make `fetch=False` in that call, and we should consider whether to pass the `fetch` option down from the CLI/DH call to this point.